var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -46,3 +47,19 @@ class LibjpegTurbo(AutotoolsPackage):
     # TODO: Implement the selection between two supported assemblers.
     # depends_on("yasm", type='build')
     depends_on("nasm", type='build')
+    depends_on('cmake', type='build', when="@1.5.90:")
+
+    @when('@:1.5.3')
+    def install(self, spec, prefix):
+        configure('--prefix=%s' % prefix)
+        make
```

Comment:
These should be `make()`, it's a function, not a statement.

Reply:
Done.